### PR TITLE
SAI 1.3 changes

### PR DESCRIPTION
**Pull request by prsunny**
Placeholder PR to migrate to SAI 1.3 headers

### CODE REVIEW

File: platform/broadcom/sai.mk [hunk 1]
```diff
@@ -1,9 +1,9 @@
-BRCM_SAI = libsaibcm_3.1.3.4-15_amd64.deb
-$(BRCM_SAI)_URL = "https://sonicstorage.blob.core.windows.net/packages/libsaibcm_3.1.3.4-15_amd64.deb?sv=2015-04-05&sr=b&sig=7CIkdlBB%2BY6Xjae0vc1QBnlAVedRkreJRhWM93SyHOI%3D&se=2032-02-29T18%3A31%3A54Z&sp=r"
+BRCM_SAI = libsaibcm_3.1.3.5_amd64.deb
```
Comment (lguohan) [inline platform/broadcom/sai.mk]: should be something like libsaibcm_3.1.3.5-1_amd64.deb

Comment (prsunny) [reply platform/broadcom/sai.mk]: Observed from the changelog, it starts without the suffix -1 as  

saibcm (3.1.3.4) unstable; urgency=medium

**Comment by lguohan** [comment]
retest this please

**Comment by qiluo-msft** [comment]
Mellanox build check failed. Please check.

**Pull request merged**